packages/i18n/data/core.json

```diff
@@ -159,9 +315,117 @@
     "developer_comment": "aria-label & tooltip for show password button",
     "string": "Show password"
   },
-  "Nimbus.ScopedSearchInput.clearButton": {
-    "developer_comment": "aria-label for clear button",
-    "string": "Clear search"
+  "Nimbus.RangeCalendar.nextMonth": {
+    "developer_comment": "aria-label for next month navigation button",
+    "string": "Next month"
+  },
+  "Nimbus.RangeCalendar.nextYear": {
+    "developer_comment": "aria-label for next year navigation button",
+    "string": "Next year"
+  },
+  "Nimbus.RangeCalendar.previousMonth": {
+    "developer_comment": "aria-label for previous month navigation button",
+    "string": "Previous month"
+  },
+  "Nimbus.RangeCalendar.previousYear": {
+    "developer_comment": "aria-label for previous year navigation button",
+    "string": "Previous year"
+  },
```

Comment:
At what point do we want to make common message files for duplicated messages, such as these between the calendars?